Dockerfile_tests

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
```

Comment:
nyeste versjon er v5 nå, burde vi gå for det?

Reply:
Denne docker-filen er kun for å kjøre tester i en container slik at man kan se hvordan det oppfører seg på github actions/linux. Kunne nok bumpet til v5, men da må jeg også installere v5 runtime og endre target i prosjektfiler. Tenker det er good for now.